src/cpp/shared/adventure/map_events.cpp

```diff
@@ -30,6 +30,11 @@ void advManager::PasswordEvent(mapCell *cell, hero *hero) {
     gpCurPlayer->barrierTentsVisited |= 1 << barrierColor;
 }
 
+void playerData::SetBarrierTentsVisited(char x) {
+	gpCurPlayer->barrierTentsVisited += 1;
+	
```

Comment:
I'm going to make you remove these trailing blank lines; might as well do it now.